tls.go

```diff
@@ -18,15 +18,19 @@ func (c *TLSConfig) TLSConfig() (*tls.Config, error) {
 		return nil, fmt.Errorf("Error while reading Root CA file: " + c.RootCAPath + " error: " + err.Error())
 	}
 
+	caCertPool := x509.NewCertPool()
+	if ok := caCertPool.AppendCertsFromPEM(rootCA); !ok {
+		return nil, fmt.Errorf("failed to append Root CA certificates from file: %s", c.RootCAPath)
+	}
+
 	interCA, err := os.ReadFile(c.IntermediateCAPath)
 	if err != nil {
-		return nil, fmt.Errorf("Error while reading Intermediate CA file: " + c.IntermediateCAPath + " error: " + err.Error())
+		fmt.Printf("Warning: Unable to read Intermediate CA file: %s, error: %v", c.IntermediateCAPath, err)
+		fmt.Println("Intermediate CA will be skipped.")
```

Comment:
If we use our log implementation, we can arrange level. Not required to add prefix `Warning`